src/main/scala/com/mozilla/telemetry/pings/Ping.scala

```diff
@@ -367,11 +379,23 @@ trait SendsToAmplitudeWithEnvironment extends SendsToAmplitude {
 
     ("user_properties" ->
       ("channel" -> meta.normalizedChannel) ~
```

Comment:
The only thing missing here is that https://github.com/mozilla/SAVANT-study-analysis/blob/ed5c4c706ad9b5ab0bd0bacc29b0daff7d00920a/AMP_CLIENTS.md mentions both channel and normalized_channel.

We are already including normalized channel as "channel". Is raw channel necessary, @SuYoungHong?

Reply:
@jklukas , good point. I suppose it's fine to exclude it. it's important for historical data, but since this study is on new data, it's fine to exclude the raw channel.